kolibri/plugins/management/assets/src/device_management/state/mutations/permissionsMutations.js

```diff
@@ -0,0 +1,13 @@
+import { PageNames } from '../../constants';
+
+export function SET_PERMISSIONS_PAGE_STATE(state, pageState) {
+  if (state.pageName === PageNames.MANAGE_PERMISSIONS_PAGE) {
```

Comment:
Our usual pattern for this is the samePageCheck within the action itself, rather than in the mutation.